contentgrid-spring-data-rest/src/test/java/com/contentgrid/spring/test/fixture/invoicing/InvoicingApplicationTests.java

```diff
@@ -1930,6 +1940,65 @@ void postMultipartContent_noPayload_http400() throws Exception {
                     mockMvc.perform(multipart(HttpMethod.POST, "/invoices/{id}/content", invoiceId(INVOICE_NUMBER_1)))
                             .andExpect(status().isBadRequest());
                 }
+
+                @Test
+                @Disabled("ACC-1216")
+                void postMultipartEntityAndContent_textPlainUtf8_http201() throws Exception {
+                    // Build mockMvc with web application context
+                    mockMvc = MockMvcBuilders.webAppContextSetup(context).build();
+                    // Create multipart file
+                    var file = new MockMultipartFile("content", "content.txt", MIMETYPE_PLAINTEXT_UTF8,
+                            UNICODE_TEXT.getBytes(StandardCharsets.UTF_8));
+
+                    mockMvc.perform(multipart("/invoices")
```

Comment:
I'm not sure the mockmvc API allows this, but is it possible to be explicit this is a POST request ?
unless I'm mistaken, a multipart/form-data requests could be a PUT too ?